tests/pt-br/cover_HassTurnOff.yaml

```diff
@@ -1,2 +1,22 @@
 language: pt-br
-tests: []
+tests:
+  - sentences:
+      - fechar a porta da garagem
+    intent:
+      name: HassTurnOff
+      slots:
+        device_class: garage
+        domain: cover
+    response: Fechado
+  - sentences:
+      - fechar as cortinas no cozinha
```

Comment:
Could you adjust the sentence to "Fechar as cortinas da cozinha"

Reply:
ok.